### PR TITLE
timestamp from s to ms

### PR DESCRIPTION
**Pull request by wenwenchenbosch**
solve #88

### CODE REVIEW

File: src/VssCommandProcessor.cpp [hunk 1]
```diff
@@ -476,7 +467,13 @@ string VssCommandProcessor::processQuery(const string &req_json,
       string path = root["path"].as<string>();
       string request_id = root["requestId"].as<string>();
       if (action == "get") {
-       //replaced, handled above
+        logger->Log(LogLevel::VERBOSE, "VssCommandProcessor::processQuery: get query  for " + path
+                    + " with request id " + request_id);
+
+        response = processGet(channel, request_id, path);
```
Comment (SebastianSchildt) [inline src/VssCommandProcessor.cpp:473]: Why using the deprecated function again?

File: src/VssCommandProcessor.cpp [hunk 2]
```diff
@@ -50,7 +50,6 @@ VssCommandProcessor::VssCommandProcessor(
   subHandler = subhandler;
   // TODO: add accessValidator as dependency
   accessValidator = std::make_shared<AccessChecker>(tokenValidator);
-  requestValidator = new VSSRequestValidator(logger);
```
Comment (SebastianSchildt) [inline src/VssCommandProcessor.cpp:53]: Where is it gone to? It is needed for verifying gen2 requests (Removing this should break all get requests)

File: src/VssCommandProcessor.cpp [hunk 3]
```diff
@@ -437,14 +435,7 @@ string VssCommandProcessor::processQuery(const string &req_json,
     string action = root["action"].as<string>();
     logger->Log(LogLevel::VERBOSE, "Receive action: " + action);
 
-    if (action == "get") {
-        response = processGet2(channel, root);
```
Comment (SebastianSchildt) [inline src/VssCommandProcessor.cpp:441]: Why disabling get requests?

**Review by SebastianSchildt — CHANGES_REQUESTED** [review]
As suspected this PR removes all V2 support

```
VSS Client> authorize ../../certificates/jwt/super-admin.json.token
{
    "TTL": 1767225599, 
    "action": "authorize", 
    "requestId": "204976604615761430279158461109939820200", 
    "timestamp": 1611167894976
}

VSS Client> getValue Vehicle/Speed
{
    "action": "get", 
    "error": {
        "message": "I can not find Vehicle/Speed in my db", 
        "number": 404, 
        "reason": "Path not found"
    }, 
    "requestId": "197203590235776989378987479460686585750", 
    "timestamp": 1611167903004
}

VSS Client> 
```

File: src/VssCommandProcessor.cpp (hunk 3 above)
Comment (wenwenchenbosch) [reply src/VssCommandProcessor.cpp:441]: strange here... I do not know what git done!

**Comment by wenwenchenbosch** [comment]
@SebastianSchildt I removed all unknown changes. I also do not know, where the changes comes from. Maybe I should always do `git status` to see, if workspace is clean or not. Somehow worked on the old `VssCommandProcessor.cpp` after checkout a new branch.

**Comment by SebastianSchildt** [comment]
Unittests seem seem to fail, is this "expected", are you working on it, or should I check in more detail?

**Comment by SebastianSchildt** [comment]
Get still broken 

```
VSS Client> connect
connect to wss://127.0.0.1:8090
Websocket connected!!
VSS Client> authorize ../../certificates/jwt/super-admin.json.token
{
    "TTL": 1767225599, 
    "action": "authorize", 
    "requestId": "271284610324058715337520418729616271945", 
    "timestamp": 1611351750942
}

VSS Client> getValue Vehicle.Speed
{
    "action": "get", 
    "path": "Vehicle.Speed", 
    "requestId": "243067668657593196351596266829391776138", 
    "timestamp": 1611351755, 
    "value": "---"
}
```

**Review by SebastianSchildt — APPROVED** [review]
I modified it so ALL (hopefully) timestamps are using the new format (get and error messages have been missing). @wenwenchenbosch  please take a quick look that it is still fine, then I will merge.

**Pull request merged**